### PR TITLE
[UGLY] notebook style layout with multiple Bots

### PR DESCRIPTION
**Pull request by nasaWelder**
still stubbed, and UGLY, but you can make the GUI switch between different bots.

### CODE REVIEW

File: simple_app.py [hunk 1]
```diff
@@ -47,6 +48,83 @@
 }
 """
 
+
+class App:
+    # adapted from notebook.py Copyright 2003, Iuri Wickert (iwickert yahoo.com)
```
Comment (needmoney90) [inline simple_app.py:53]: Whats the license on this, out of curiosity?

**Review by needmoney90 — COMMENTED** [review]
Im not super up to speed on how the various licenses fit together, but im fairly sure GPL projects can include most things (the other way around is harder). Id like to know the license before approving, just want to take a quick look to see if we're cool legally.

**Comment by needmoney90** [comment]
After googling around a bit, I'm finding references to the copyright text, but no actual definition of what particular license this code is under. As this is a fairly generic usage of tkinter, and I can't easily find the desires of the original author, I'm going to merge it. Since this code is GPL'd, I don't think there will be any issues, but I'm trying to be careful not to add incompatibly licensed code to this repo.

**Comment by nasaWelder** [comment]
@livinginformation ok, sorry I wasn't even thinking about that issue. I was going to do this exact same functionality, but when google spat it out, I gobbled it up. I can always recreate it in a different way.

sorry for the hassle

**Comment by needmoney90** [comment]
No worries at all, just trying to be proactive.

**Pull request merged**